act-rules-format.bs

```diff
@@ -156,7 +150,7 @@ For each accessibility requirement in the mapping, an ACT Rule MUST indicate wha
   </ul></blockquote>
 </div>
 
-ACT Rules can be used to test accessibility requirements that are not part of a W3C accessibility standard, such as [[WAI-ARIA]] requirements, or a testing methodology like [RGAA 3](https://disic.github.io/rgaa_referentiel_en/criteria.html). An ACT Rule MUST indictate whether or not the [=accessibility requirement=] it maps to is required for conformance in its [=accessibility requirements document=]. Examples of accessibility requirements that are not required for conformance are WCAG sufficient techniques, or a company style guide which includes both requirements and optional "best practices". That distinction between what is required and what is optional has to be made clear.
+ACT Rules can be used to test accessibility requirements that are not part of a W3C accessibility standard, such as [Hypertext Markup Language](https://www.w3.org/TR/html/) [[HTML]] requirements, or a testing methodology like [RGAA 3 2016](https://disic.github.io/rgaa_referentiel_en/criteria.html). An ACT Rule MUST indictate whether or not the [=accessibility requirement=] it maps to is required for conformance in its [=accessibility requirements document=]. Examples of accessibility requirements that are not required for conformance are WCAG sufficient techniques, or a company style guide which includes both requirements and optional "best practices". The distinction between what is required and what is optional has to be made clear.
```

Comment:
"The distinction ... has to be made clear" - should that be a MUST requirement?

Reply:
editorial